### ISSUE TITLE
[Bug] does not work with remapped keys

### ISSUE THREAD

**Issue by B4rc1** [ja-ford/delaytrain.nvim#3]
Hi there, I really like the idea of the plugin and wanted to try it out for myself.
On [my keyboard](https://configure.zsa.io/moonlander/layouts/DA5gq/latest/0) layout, `jkhl` are scattered around the keyboard, so i remap keys so that I move with `dtrn` like this:
```vimscript
noremap h d
noremap d h
noremap k r
noremap r k
noremap j t
noremap t j
noremap l n
noremap n l

noremap H D
noremap D H
noremap K R
noremap R K
noremap J T
noremap T J
noremap L N
noremap N L
```

now when using this plugin either I do this:
```lua
    require('delaytrain').setup {
      delay_ms = 1000,  -- How long repeated usage of a key should be prevented
      grace_period = 3, -- How many repeated keypresses are allowed
      keys = {          -- Which keys (in which modes) should be delayed
        ['nv'] = {
          'd', 't', 'r', 'n', -- movement
          'b', 'w', 'B', 'W'  -- word actions
        },
        ['nvi'] = {'<Left>', '<Down>', '<Up>', '<Right>'},
      },
```
Resulting in my mapping getting undone (can no longe move with for example `t`, waiting for another key to execute the regular `t` command)

or:
```lua
use {
  'ja-ford/delaytrain.nvim',
  config = function ()
    require('delaytrain').setup {
      delay_ms = 1000,  -- How long repeated usage of a key should be prevented
      grace_period = 3, -- How many repeated keypresses are allowed
      keys = {          -- Which keys (in which modes) should be delayed
        ['nv'] = {
          'j', 'k', 'h', 'l', -- movement
          'b', 'w', 'B', 'W'  -- word actions
        },
        ['nvi'] = {'<Left>', '<Down>', '<Up>', '<Right>'},
      },
    }
  end
}
```
This time my normal movement keys `dtrn` do not get overridden, meaning I can move like normal but no delay is applied to the keys, instead my `jkhl` mapping gets overridden and I can move with them aswell but with delay applied.

**Comment by kohane27** [comment]
Thank you for raising this issue! I have the same issue too. I'm trying to wane myself off of J (remapped to `5j`)  and K (remapped to `5k`) and realized this plugin doesn't work. Look forward to seeing remap support!

**Comment by ja-ford** [comment]
Hi everyone, thanks to @al-ce this issue has been fixed and tagged as the 1.0.1 release

You should now be able to include remapped keys in your config without anything extra. At the moment this does not work if the key is mapped to a function. I deemed this less critical but if this is an issue for anyone else I can try and loop back around to it.

I'll close this for now but feel free to let me know if this isn't working as expected.
